test/statistics/stats_test.cpp

```diff
@@ -365,11 +369,17 @@ TEST_F(StatsTest, PerQueryStatsTest) {
   LOG_INFO("Table created!");
 
   auto backend_context = stats::BackendStatsContext::GetInstance();
+  // Get a query param object
+  std::shared_ptr<uchar> type_buf;
+  std::shared_ptr<uchar> format_buf;
+  std::shared_ptr<uchar> val_buf;
+  auto query_params =
+      StatsTestsUtil::GetQueryParams(type_buf, format_buf, val_buf);
```

Comment:
Is it possible to test the serialization and deserialization of query_params? It seems to be hard to do it with ctest. But is it feasible to implement a simple python validator to do that?